### PR TITLE
Move the add-address button to the bottom of the address list

### LINKED ISSUE #114: Move Add-Address button from top-right corner to below address list
Instead of the plus button to add a new address being at the top right of the "Addresses" card, it should be at the bottom below the address list, in the place where the new address will then pop up:

image

### PR DESCRIPTION
**Pull request by sisou**
Resolves #114.

image

### CODE REVIEW

File: src/elements/x-accounts/x-accounts.css [hunk 1]
```diff
@@ -115,6 +115,32 @@ x-account x-amount {
...
+
+x-accounts .add-address:hover,
+x-accounts .add-address:focus {
+    opacity: 1;
+}
+
+x-accounts .add-address [identicon-plus] {
+    display: block;
+    width: 60px;
+    height: 60px;
```
Comment (danimoh) [inline src/elements/x-accounts/x-accounts.css]: I think it would look better if it would be the same size as the identicons.

Comment (sisou) [reply src/elements/x-accounts/x-accounts.css]: I tried that, and it looked massive! The designs for the new Safe use much smaller identicons, so it works there, but in the current account list I think it's too big.

Comment (sisou) [reply src/elements/x-accounts/x-accounts.css]: With the new Safe layout, the address list will change to the Vue component (which also needs to be adapted with this new icon), and I think then we can make it the same size.

Comment (julianbauer) [reply src/elements/x-accounts/x-accounts.css]: Smaller size looks okay imo

**Pull request merged**